<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Caesium</class>
 <widget class="QMainWindow" name="Caesium">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Helvetica</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Caesium</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #F8F9FA;</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="header_Label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Helvetica</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #FFFFFF;
border-bottom: 2px solid #F2F2F2;</string>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="font">
       <font>
        <family>Helvetica</family>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>7</number>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="font">
        <font>
         <family>Helvetica</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="opaqueResize">
        <bool>true</bool>
       </property>
       <property name="handleWidth">
        <number>7</number>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="font">
         <font>
          <family>Helvetica</family>
         </font>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="topMargin">
          <number>12</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QTreeWidget" name="list_TreeWidget">
           <property name="font">
            <font>
             <family>Helvetica</family>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QTreeWidget {
	background-color: #FFFFFF;
	outline: none;
	border: none;
}

QHeaderView {
	background-color: #FFFFFF;
    text-transform: uppercase;
}
QHeaderView::section {
    height: 40px;
	background-color: #FFFFFF;
	border: none;
	line-height: 15px;
	border-bottom: 2px solid #F2F2F2;
}

QTreeView::item {
	height: 35px;
	border-bottom: 1px solid #F2F2F2;
}

QTreeView::item:hover {
	background-color: #F2F2F2;
}
</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>true</bool>
           </property>
           <property name="indentation">
            <number>0</number>
           </property>
           <property name="itemsExpandable">
            <bool>false</bool>
           </property>
           <column>
            <property name="text">
             <string/>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Name</string>
            </property>
            <property name="font">
             <font>
              <family>Helvetica</family>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Size</string>
            </property>
            <property name="font">
             <font>
              <family>Helvetica</family>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Resolution</string>
            </property>
            <property name="font">
             <font>
              <family>Helvetica</family>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Saving</string>
            </property>
            <property name="font">
             <font>
              <family>Helvetica</family>
              <italic>false</italic>
             </font>
            </property>
           </column>
           <item>
            <property name="text">
             <string/>
            </property>
            <property name="text">
             <string>Test1</string>
            </property>
            <property name="text">
             <string>0.00Kb</string>
            </property>
            <property name="text">
             <string>1920x1024</string>
            </property>
            <property name="text">
             <string>-80%</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string/>
            </property>
            <property name="text">
             <string>Test2</string>
            </property>
            <property name="text">
             <string>0.00Kb</string>
            </property>
            <property name="text">
             <string>1920x1024</string>
            </property>
            <property name="text">
             <string>-90%</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string/>
            </property>
            <property name="text">
             <string>Test3</string>
            </property>
            <property name="text">
             <string>0.00Kb</string>
            </property>
            <property name="text">
             <string>1920x1024</string>
            </property>
            <property name="text">
             <string>-89%</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string/>
            </property>
            <property name="text">
             <string>Test4</string>
            </property>
            <property name="text">
             <string>0.00Kb</string>
            </property>
            <property name="text">
             <string>1920x1024</string>
            </property>
            <property name="text">
             <string>-10%</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="font">
         <font>
          <family>Helvetica</family>
         </font>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>12</number>
         </property>
         <property name="rightMargin">
          <number>12</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QFrame" name="options_Frame">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #FFFFFF;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>42</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>42</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border-bottom: 2px solid #F2F2F2;</string>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_3">
       <property name="font">
        <font>
         <family>Helvetica</family>
        </font>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="leftMargin">
         <number>12</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>12</number>
        </property>
        <property name="bottomMargin">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="originalImagePreview_Label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>150</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #FFFFFF;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="compressedImagePreview_Label">
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #FFFFFF;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
    </font>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
